### PR TITLE
Button styling tweaks

### PR DESCRIPTION
**Pull request by styrelius**
## What?

Changes to make our `Button` component slightly more in line with the buttons in the [Web UI Kit Figma](https://www.figma.com/file/zlX4FcM5dMRz9stsCNQJXB/Web-UI-Kit?node-id=1441%3A2932):
- More horizontal padding and less vertical, for buttons that don't use the `lg` prop 
- New 'disabled' style

## Why?

💅 

<hr />

_Please note that I skipped fixing the `:hover` and `:focus` style since that would require remaking the entire component and it doesn't make sense for me to spend that much time on this now_

## Screenshots / recordings 

**New** button, enabled:
image

**Current** button, enabled:
image

**New** button, `fullWidth`, enabled:
image

**Current** button, `fullWidth`, enabled:
image

**New** button, `fullWidth`, `lg`, disabled:
image

**Current** button, `fullWidth`, `lg`, disabled:
image

### CODE REVIEW

File: src/client/components/buttons.tsx [hunk 1]
```diff
@@ -31,13 +31,9 @@ export const Button = styled.button<ButtonProps>`
...
   width: ${(props) => (props.fullWidth ? '100%' : 'auto')};
   padding: ${(props) =>
-    props.size === 'lg' ? `1.5rem 2.5rem` : `0.875rem 1.5rem`};
-  background: ${(props) =>
-    props.disabled
-      ? color(props.background ?? colorsV3.gray700)
-          .lighten(0.1)
-          .toString()
-      : props.background ?? colorsV3.gray900};
+    props.size === 'lg' ? `1.5rem 2.5rem` : `0.75rem 2rem`};
```
Comment (guilhermespopolin) [inline src/client/components/buttons.tsx:34]: Nit: replace template strings with regular ones since we don't actually need them here.

**Review by guilhermespopolin — DISMISSED** [review]
💅🏻

File: src/client/components/buttons.tsx (hunk 1 above)
Comment (styrelius) [reply src/client/components/buttons.tsx:34]: Done 👍

**Review by guilhermespopolin — APPROVED** [review]
Nice refactoring :)

**Pull request merged**